GoRogue/MapGeneration/Region.cs

```diff
@@ -178,302 +134,67 @@ public Region(Point northWest, Point northEast, Point southEast, Point southWest
         /// Returns a string detailing the region's corner locations.
         /// </summary>
         public override string ToString()
-            => $"Region: NW{NorthWestCorner}=> NE{NorthEastCorner}=> SE{SouthEastCorner}=> SW{SouthWestCorner}";
+        {
+            var answer = "Region: ";
+            foreach (var corner in Area.Corners)
+                answer += $"{corner} => ";
+            return answer;
```

Comment:
```suggestion
            var answer = new StringBuilder("Region: ");
            foreach (var corner in Area.Corners)
                answer.Append($"{corner} => ");
            return answer.ToString();
```
Switched to `StringBuilder` for minor optimization for things with lots of corners.

Additionally, is there a reason to have this `ToString()` here as opposed to on `PolygonArea`?